src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -442,11 +440,11 @@ void StridedSlice::StridedSliceCommonExecutor::paramsInitialization(const Stride
     params.attrs.beginDims = srcMemory[attrs.BEGIN_ID]->getShape().getStaticDims();
     params.attrs.endDims = srcMemory[attrs.END_ID]->getShape().getStaticDims();
     if (params.attrs.beginDims.size() != 1)
-        OPENVINO_THROW(errorPrefix, "should have begin vector with 1 dimension");
+        OPENVINO_THROW("should have begin vector with 1 dimension");
```

Comment:
Please rework the error messages enhancing them with at least the name of the executor class and changing the wording, or, alternatively, generate the error prefix at the runtime using the node name and type. Here and below.

Reply:
Done